public/config/README_configuration.md

```diff
@@ -29,81 +29,65 @@ The configuration file has four main sections and then several key-value pairs.
-* screen
-* service - a string defining the associated ROS service to call.
-* serviceAttribute - which attribute(s) of the serviceType are populated with data from the UI.
-* serviceType - a string defining the ROS interface for the service.
-* smalltick
-* step
-* suffix - appended to the msgAttribute value before displaying
-* trueText - what's displayed when msgAttribute is a boolean and true
-* textColor
-* textColor2
-* top - a string in the format "#px" interpreted as the number of pixels from
-        the top of the parent element.
-* topic - a string defining the associated ROS topic to publish or subscribe.
-* topicDirection - a string with one of "publish" or "subscribe"
-* topicAttribute - which attribute(s) of the topicType, like 'header.stamp.sec', are populated
-                   with data from the UI or used to update the UI. multiple attributes are
-                   separated by semi-colon(s).
-* topicType - a string defining the ROS interface for the topic, such as std_msgs/msg/String.
-* type - a string describing the widget type. must exist as an HTML DIV "id"
-         attribute in the HTML page.
-* useAxis
-* useButton
-* useGamepad
-* usekey_Decrease
-* usekey_down
-* usekey_hotkey
-* usekey_Increase
-* usekey_left
-* usekey_right
-* useKeys
-* usekey_up
-* useLeft - SUPERFLUOUS - if "true" use the "left" value to position the widget and ignore the right
-            value. if "false" then do the opposite.
-* useROS
-* useTop - SUPERFLUOUS - if "true" use the "top" value to position the widget and ignore the bottom
-           value. if "false" then do the opposite.
-* vertical
-* w - the width of the widget as "#px"
-* xScaleFactor, yScaleFactor - floating point value used with the joystick to scale
-                               the x and y values.
+#### widget properties
 
+[CSS reference](https://www.w3schools.com/cssref/index.php)
+[widget lib](https://jqueryui.com/widget/)
+[position lib](https://jqueryui.com/position/)
+[button lib](https://jqueryui.com/button/)
+[slider lib](https://jqueryui.com/slider/)
+
+the widget properties have the following sections:
+        root: top level properties generic to all widgets
+        position: properties for positioning the widget
+        format: properties for formatting the display of the widget content
+        data: topics and services associated with the widget
+
+* id: unique integer for each widget
+* type: string defining the widget type
+* label: string displayed on the widget header
+
+* position.my: string defining x,y of widget to position with respect to the parent
+* position.at: string defining x,y of parent to position widget
+
+* format.prefix: (value) string prepended to the value before displaying
+* format.suffix: (value) string appended to the value before displaying
+* format.precision: (value) number of decimal places to display for values 
+
+* format.trueText: (state) string displayed when state is true
```

Comment:
I find this addition useful too, to clearly describe for which this attribute is relevant.
For consistency, if it's described as "Slider" in the list above, which she keep the case consistent here.